### PR TITLE
docs: Improve section 'Why did you write Hugo?'

### PR DESCRIPTION
**Pull request by MarkDBlackwell**
"The message" IMO sells better, when one signs one's name.

Includes other polishing.

### CODE REVIEW

File: docs/content/overview/introduction.md [hunk 1]
```diff
@@ -154,30 +154,33 @@ pages.
 
 I wrote Hugo ultimately for a few reasons. First, I was disappointed with
 WordPress, my then website solution. It rendered slowly. I couldn't create
-content as efficiently as I wanted to and needed to be online to write
-posts. The constant security updates and the horror stories of people's
+content as efficiently as I wanted to and I needed to be online to write
```
Comment (moorereason) [inline docs/content/overview/introduction.md]: If you're going to add `I` here, you need to add a comma before `and`; however, I think the extra `I` is unnecessary.

File: docs/content/overview/introduction.md [hunk 2]
```diff
@@ -154,30 +154,33 @@ pages.
...
-I looked at existing static site generators like [Jekyll][], [Middleman][] and [nanoc][].
-All had complicated dependencies to install and took far longer to render
-my blog with hundreds of posts than I felt was acceptable. I wanted
-a framework to be able to get rapid feedback while making changes to the
-templates, and the 5+-minute render times was just too slow. In general,
-they were also very blog minded and didn't have the ability to have
-different content types and flexible URLs.
+I looked at the existing static site generators like
+[Jekyll][], [Middleman][] and [Nanoc][].
+All had complicated dependencies to install, and took far longer to render
```
Comment (moorereason) [inline docs/content/overview/introduction.md]: The added comma before `and` is unnecessary.

**Comment by MarkDBlackwell** [comment]
Improved the text, partly based on your comments.

**Comment by digitalcraftsman** [comment]
Hello again,

kudos for your continues improvements of the documentation :+1: I've merged your pull request as 8474b9defeab3cb1ea8c697bbe16ba354b578c94.

Cheers,
Digitalcraftsman

**Pull request closed without merging**